llvm/docs/CommandGuide/llvm-symbolizer.rst

```diff
@@ -216,6 +224,12 @@ OPTIONS
   This can be used to perform lookups as if the object were relocated by the
   offset.
 
+.. option:: --approximate-line
```

Comment:
Maybe this should be called "ignore zero line entries" or something like that? It might more accurately describe the behavior, without having to haggle about "what kind of approximation is used"?